src/utils/validationUtil.ts

```diff
@@ -42,6 +42,22 @@ export class ValidationUtil {
       payload.vehicleType,
       payload.testStatus,
     );
+
+    if (this.isIvaTest(payload.testTypes)) {
+      const allFailWithoutDefects = payload.testTypes.every(
```

Comment:
Is it worth busting this logic out into a separate validationUtils method like the others. For example:

```
if(!ivaFailedTestHasRequiredFields(payload.testTypes)) {
    throw new models.HttpError( ... etc
}
```

Obviously not too worried about what the name is.

Reply:
What would be the advantage of busting it?

Reply:
Like the others, a little bit cleaner and the validation logic is in Validation Utils with all the other validation logic.